gridmap/job.py

```diff
@@ -238,6 +248,12 @@ def native_specification(self):
             ret += " -q {}".format(self.queue)
         if self.gpu:
             ret += " -l gpu={}".format(self.gpu)
+        if self.h_vmem:
+            ret += " -l h_vmem={}".format(self.h_vmem)
+        if self.h_rt:
+            ret += " -l h_rt={}".format(self.h_rt)
+        if self.resources:
+            ret += " " + " ".join([" -l {}".format(x) for x in self.resources])
```

Comment:
Is there a reason why this shouldn't just be?
```suggestion
            ret += "".join([" -l {}".format(x) for x in self.resources])
```

Reply:
No, I believe you're correct. Let me commit the suggestion.